benchmarks/utils/update_artifacts.py

```diff
@@ -0,0 +1,86 @@
+import argparse
+import os
+import shutil
+
+BENCHMARK_REPORT_PATH = "/tmp/ts_benchmark"
+BENCHMARK_ARTIFACTS_PATH = "/tmp/ts_artifacts"
+BENCHMARK_REPORT_FILE = "ab_report.csv"
+WINDOW_LEN = 8
+WINDOW_START = 0
+
+# Copy BENCHMARK_REPORT_FILE to artifacts
+def copy_benchmark_reports(input, output):
+
+    for dir in os.listdir(input):
+        if os.path.isdir(os.path.join(input, dir)):
+            new_dir = os.path.join(output, dir)
+            os.makedirs(new_dir, exist_ok=True)
+            shutil.copy(os.path.join(input, dir, BENCHMARK_REPORT_FILE), new_dir)
+
+
+# Create artifacts for a period of rolling WINDOW_LEN-1 reports
+def update_artifacts(args):
+
+    finished_copying = False
+    os.makedirs(args.output, exist_ok=True)
+    if not os.listdir(args.output):
+        new_dir = os.path.join(args.output, str(WINDOW_START))
+        print(
+            f"There are no artifacts. A new package needs to be created starting at {new_dir}"
+        )
+        # shutil.copytree(args.input, new_dir)
+        copy_benchmark_reports(args.input, new_dir)
+
+    else:
+        list_dirs = sorted(map(lambda x: int(x), os.listdir(args.output)))
+        for i, dir in enumerate(list_dirs):
+
+            if i != dir:
```

Comment:
Not sure I follow what this line is trying to do   `list_dirs = sorted(map(lambda x: int(x), os.listdir(args.output)))` why are we converting string names to ints and sorting?

EDIT: Oh I see because the prefix is a number - I'd make that clearer in the variable names

Reply:
Added more comments to make the logic clearer